programs/manifest/src/state/market.rs

```diff
@@ -1246,16 +1352,24 @@ impl<
             get_free_address_on_market_fixed_for_ask_order(fixed, dynamic)
         };
 
-        let resting_order: RestingOrder = RestingOrder::new(
+        let mut resting_order: RestingOrder = RestingOrder::new(
```

Comment:
This change would seem dangerous - we don't get any coverage on resting orders from FV?

Reply:
What is dangerous?
What coverage are you referring to?